README.md

```diff
@@ -13,15 +16,35 @@
 * [Production](https://michigan-benefits-production.herokuapp.com)
 * [Prototype demo](https://michigan-benefits-prod.herokuapp.com/)
 
-## Driving Against MI Bridges
+## Documentation
+
+Documentation is contained in Markdown files within the repository
+
+| File          | Purpose |
+| ------------- | -----------|
+| [README.md](README.md) | This root document |
+| [CONTRIBUTING.md](CONTRIBUTING.md) | How we write and test code for this application |
+| [DESIGN-README.md](DESIGN-README.md) | How we design the application |
+| [DEPLOYING.md](DEPLOYING.md) | How to deploy code to production servers |
+| [LICENSE.md](LICENSE.md) | Our software license |
+
+
+## Driving Applications
```

Comment:
it feels like this could be a part of `CONTRIBUTING` since it's something that is mostly dev-specific